mappings/net/minecraft/world/biome/layer/ScaleLayer.mapping

```diff
@@ -1,3 +1,7 @@
 CLASS net/minecraft/class_3656 net/minecraft/world/biome/layer/ScaleLayer
 	METHOD method_15853 sample (Lnet/minecraft/class_3628;IIII)I
 		ARG 1 context
+		ARG 2 tl
+		ARG 3 tr
+		ARG 4 bl
+		ARG 5 br
```

Comment:
Back to intermediary v1! (joke apart, I think javadocs would be really appreciated to explain these params)

Reply:
Looks like top/bottom left/right